scaffolder-skeletons/catalog-info-skeleton/catalog-info.yaml

```diff
@@ -1,22 +1,28 @@
+{%- if values.componentName %}
```

Comment:
`componentName` is used in multiple places.
To avoid duplicating the same `if else` block, I have stored `name` in a new variable.